sdks/go/test/integration/integration.go

```diff
@@ -123,6 +123,8 @@ var sparkFilters = []string{
 	// The trigger and pane tests uses TestStream
 	"TestTrigger.*",
 	"TestPanes",
+	// [BEAM-13921]: Spark doesn't support side inputs to executable stages
+	"TestDebeziumIO_BasicRead",
```

Comment:
Thanks for doing this! Just for my edification, how was this failure introduced - did we add the integration test and its never worked on spark, or did we add a feature that then broke this test on Spark?

I ask because I'm wondering if it would be worthwhile to introduce a policy that when the integration tests are changed, all post commit tests like this run automatically (or if we already have a policy like that and we just missed this). I much prefer robots checking stuff like this to humans remembering to run everything

Reply:
So, the tests listed in "integration.go" are only covering Go integration tests. So, we added the test when we added the DebeziumIO transform wrapper to the SDK.

In this case, we probably should have changes to the .../go/test/integration/... directory to execute all the post commits as part of the PR, to ensure they're run for all the runners on addition or change. 

We don't usually do this for arbitrary SDK changes, because they're much heavier weight than the various unit tests, and similarly, the Same Suite of tests is run against the Python Portable Runner as the "GoPortable Precommit", and cases where missing Runner features are tickled by the Go SDK itself, are rare. 

But that's why this got hit I guess: The issue is triggered from Java SDK code, not Go SDK code.

To sum up, this is the post commits WAI, but we should trigger them eagerly for PRs that are adding new tests to the suite.